internal/kms/keyprotect.go

```diff
@@ -23,14 +23,15 @@ import (
 	"fmt"
 
 	"github.com/ceph/ceph-csi/internal/util/k8s"
-
+	
+	"k8s.io/klog/v2"
 	kp "github.com/IBM/keyprotect-go-client"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	kmsTypeKeyProtectMetadata = "kp-metadata"
-
+	kmsTypeKeyProtectMetadata    = "ibmkeyprotect"
+	kmsTypeKeyProtectMetadataOld = "kp-metadata"
```

Comment:
this needs to be taken care  at `getSecrets` function or else the return message in getSecrets seems wrong.

Reply:
This is addressed in a different way now , ptal @Madhu-1